tests/unit/broccoli/ember-app/import-test.js

```diff
@@ -0,0 +1,224 @@
+'use strict';
+
+const co = require('co');
+const fs = require('fs-extra');
+const broccoliTestHelper = require('broccoli-test-helper');
+const expect = require('chai').expect;
+
+const EmberApp = require('../../../../lib/broccoli/ember-app');
+const MockCLI = require('../../../helpers/mock-cli');
+const Project = require('../../../../lib/models/project');
+
+const buildOutput = broccoliTestHelper.buildOutput;
+const createTempDir = broccoliTestHelper.createTempDir;
+
+describe('EmberApp.import()', function() {
+  let input, cwd;
+
+  beforeEach(function() {
+    cwd = process.cwd();
+    return createTempDir().then(tempDir => (input = tempDir));
+  });
+
+  afterEach(function() {
+    process.chdir(cwd);
+    delete process.env.EMBER_ENV;
+    return input.dispose();
+  });
+
+  function createApp(options) {
```

Comment:
Can you make an issue to refactor a shared create app helper, so we can remove the duplicates?